tools/picard/picard_AddCommentsToBam.xml

```diff
@@ -1,4 +1,4 @@
-<tool name="AddCommentsToBam" id="picard_AddCommentsToBam" version="1.126.0">
+<tool name="AddCommentsToBam" id="picard_AddCommentsToBam" version="1.135">
```

Comment:
That makes it impossible to update single tools withing this tool “package”. Can't find where I've seen that before, but I would say, this should be discussed first somewhere else.